spreadsheets/transit.csv

```diff
@@ -1,5 +1,7 @@
 kind,sort_key
-tram,415
-light_rail,415
+light_rail,417
+tram,416
+funicular,415
+monorail,414
```

Comment:
This range should start at 415, there's only a couple slots at 414 and below before there are collisions with bridges. The room to grow is up towards 425. So monorail 415, funicular 416, and on up.